test-suite/output/Succeed.out

```diff
@@ -0,0 +1,9 @@
+0
```

Comment:
Perhaps it would be better to have a message "The command has indeed succeeded with message:" since otherwise a user might miss a Succeed command.

The example I have in mind is `Definition foo := 0` which would print a message such as `foo is defined` when `Succeed` absorbs the side effect. Having the message for succeed would be clearer perhaps.

Reply:
There is such a message in interactive mode.
output/ tests are run with coqc which doesn't print it.

Reply:
Thanks, I missed that.